.flake8

```diff
@@ -15,3 +15,4 @@ per-file-ignores =
     app/main/__init__.py : E402, F401
     app/status/__init__.py : E402, F401
     app/main/views/*.py : C901
+    app/main/helpers/validation.py : C901
```

Comment:
Can we add the flask ignore to the particular method rather than the file as a whole?

Reply:
Tbh I might just bump up the complexity limit from 7 to 8 and get rid of the ignore.

Reply:
Haha